pptx_blueprint/__init__.py

```diff
@@ -52,6 +57,13 @@ def save(self, filename: __Pathlike):
 
         Args:
             filename (path-like): file name or path
+
+        Raises:
+            TemplateOvewriteError
         """
-        # TODO: make sure that the user does not override the self._template_path
-        pass
\ No newline at end of file
+        if pathlib.Path(filename).absolute() == pathlib.Path(self._template_path).absolute():
+            raise TemplateOvewriteError(
+                f'The specified save path ({filename}) is equal to the path of the template file.' \
+                'The template should not be overwritten.'
+            )
+        self._presentation.save(filename)
```

Comment:
We should check whether _presentation.save is throwing any errors (e.g., if the file is not writeable). If so, we should catch these and rethrow a custom exception (derived from the designated PptxBlueprintException that should be the parent class of all our exceptions). Or is there any other error handling in _presentation.save()?

Reply:
I'm sorry I make you do this. I just took a quick look into the package. So frustrating...

Reply:
According to @timhoffm, we could leave that like that. `OSError`s are basically good enough in the library level, as long as we don't provide a CLI.

Reply:
At least in the current situation it seems just not worth it to do exhaustive error handling. I would not want this to block us, so I'm in line with @timhoffm .